pkg/agent/core/ngt/service/ngt_stateful_test.go

```diff
@@ -515,8 +516,7 @@ var (
 			result commands.Result,
 		) *gopter.PropResult {
 			rc := result.(*resultContainer)
-
-			if rc.err != nil {
+			if rc.err != nil && !errors.Is(rc.err, errors.ErrEmptySearchResult) {
```

Comment:
Please check `rc.err` is `ErrEmptySearchResult` when all of the idA, idB, idC are not indexed.